explanation/security/charms.md

```diff
@@ -27,3 +27,33 @@ When Anbox Cloud is deployed without the use of an external CA, the charms will
 ### Packages used
 
 * [cryptography from PyPI](https://pypi.org/project/cryptography/)
+
+## Juju secrets
+
+Confidential data such as private keys, API tokens, or credentials managed by the charm or passed through relations between charms is securely handled using [Juju secrets](https://documentation.ubuntu.com/juju/3.6/reference/secret/) if you deploy Anbox Cloud with Juju 3.0 or later. You can use the following command to view the secrets used by charms:
+
+    ```
+    juju list-secrets
+    ```
```

Comment:
This doesn't render correctly, this should be either:

````suggestion
```
juju list-secrets
```
````

Or:

```suggestion

    juju list-secrets

```

Reply:
Thanks for pointing this out.
Fixed that.